Adyen.EcommLibrary.Test/CheckoutTest.cs

```diff
@@ -3,21 +3,35 @@
 using System;
 using System.Collections.Generic;
 using System.Text;
-using static Adyen.EcommLibrary.Model.Checkout.PaymentResponse;
+using static Adyen.EcommLibrary.Model.Checkout.PaymentsResponse;
 
 namespace Adyen.EcommLibrary.Test
 {
     [TestClass]
     public class CheckoutTest : BaseTest
     {
+        [TestMethod]
+        public void CheckoutEndpointTest()
+        {
+            var config = new Config();
+            var client = new Client(config);
```

Comment:
Can we  use the same contractors as java ? https://docs.adyen.com/developers/development-resources/libraries#usingthelibrary1

Reply:
it is added at [commit](https://github.com/Adyen/adyen-dotnet-api-library/pull/39/commits/ccd837d91e7e190eeab5acd405d97e4e9b60ba3f)

Reply:
great!